Jenkinsfile

```diff
@@ -16,13 +16,16 @@ def releaseTag = new com.socrata.ReleaseTag(steps, service)
 pipeline {
   options {
     ansiColor('xterm')
+    buildDiscarder(logRotator(numToKeepStr: '50'))
+    disableConcurrentBuilds(abortPrevious: true)
+    timeout(time: 20, unit: 'MINUTES')
   }
   parameters {
-    booleanParam(name: 'RELEASE_BUILD', defaultValue: false, description: 'Are we building a release candidate?')
-    booleanParam(name: 'RELEASE_DRY_RUN', defaultValue: false, description: 'To test out the release build without creating a new tag.')
-    string(name: 'RELEASE_NAME', defaultValue: '', description: 'For release builds, the release name which is used for the git tag and the deploy tag.')
     string(name: 'AGENT', defaultValue: 'build-worker', description: 'Which build agent to use?')
     string(name: 'BRANCH_SPECIFIER', defaultValue: 'origin/main', description: 'Use this branch for building the artifact.')
+    booleanParam(name: 'RELEASE_BUILD', defaultValue: false, description: 'Are we building a release candidate?')
```

Comment:
Later, once these jobs are modified to allow for hotfixes, I think this parameter should be reframed as build purpose, as expressed in RMS.

Reply:
Since hotfixes are triggered automatically based on the branch name, we can't set a parameter for those builds.  That is why there is a `var` that's used to set `isHotfix`.